docs/sphinx/_ext/dagster-sphinx/dagster_sphinx/docstring_flags.py

```diff
@@ -82,7 +82,13 @@ class inline_flag(nodes.Inline, nodes.TextElement):
 
 def visit_inline_flag(self, node: inline_flag):
     flag_type = node.attributes["flag_type"]
-    html = f'<span class="flag {flag_type}">{flag_type}</span>'
+    html = f"""
+    <span class="flag {flag_type}">
+      <span class="hidden">(</span>
+      {flag_type}
+      <span class="hidden">)</span>
```

Comment:
The hidden parens are picked up by Docs search scraping.

Reply:
Mind adding this as a comment in the source